### ISSUE TITLE
Formio npm install fails to build: npm ERR! 404 Not Found: flatmap-stream@0.1.2

### ISSUE THREAD

**Issue by igdianov** [formio/formio#697]
```
Step 1/14 : FROM node:10.13.0-alpine                                                                                                                                           
 ---> e35872f034fd                                                                                                                                                             
Step 2/14 : RUN apk add sudo                                                                                                                                                   
 ---> Using cache                                                                                                                                                              
 ---> 2044ba80cf7d                                                                                                                                                             
Step 3/14 : RUN apk update &&         apk add             expect             git             g++             jq             make             python                            
 ---> Using cache                                                                                                                                                              
 ---> e36e096a7320                                                                                                                                                             
Step 4/14 : ENV FORMIO_VERSION=v1.37.4         FORMIO_CLIENT_VERSION=master                                                                                                    
 ---> Using cache                                                                                                                                                              
 ---> c3628a76f321                                                                                                                                                             
Step 5/14 : RUN   git clone -b $FORMIO_VERSION https://github.com/formio/formio.git /app/                                                                                      
 ---> Using cache                                                                                                                                                              
 ---> 7d7be746d1d2                                                                                                                                                             
Step 6/14 : RUN   git clone -b $FORMIO_CLIENT_VERSION https://github.com/formio/formio-app-formio.git /app/client                                                              
 ---> Running in 58f39be29192                                                                                                                                                  
Cloning into '/app/client'...                                                                                                                                                  
Removing intermediate container 58f39be29192                                                                                                                                   
 ---> b1de12cd4485                                                                                                                                                             
Step 7/14 : RUN   cd /app &&            npm install                                                                                                                            
 ---> Running in 60b790b14164                                                                                                                                                  
npm WARN tar ENOENT: no such file or directory, open '/app/node_modules/.staging/eslint-scope-9052cb51/package.json'                                                           
npm WARN tar ENOENT: no such file or directory, open '/app/node_modules/.staging/eslint-scope-9052cb51/CHANGELOG.md'                                                           
npm WARN tar ENOENT: no such file or directory, open '/app/node_modules/.staging/eslint-scope-9052cb51/LICENSE'                                                                
npm WARN tar ENOENT: no such file or directory, open '/app/node_modules/.staging/eslint-scope-9052cb51/README.md'                                                              
npm WARN tar ENOENT: no such file or directory, open '/app/node_modules/.staging/extsprintf-a7e4bffe/jsl.node.conf'                                                            
npm WARN tar ENOENT: no such file or directory, open '/app/node_modules/.staging/async-67304d10/dist/async.min.js'                                                             
npm WARN tar ENOENT: no such file or directory, open '/app/node_modules/.staging/fast-json-patch-c7b41b16/dist/fast-json-patch.min.js'                                         
npm WARN tar ENOENT: no such file or directory, open '/app/node_modules/.staging/i18next-2b852b7b/i18next.min.js'                                                              
npm WARN tar ENOENT: no such file or directory, open '/app/node_modules/.staging/iconv-lite-046664fa/encodings/tables/eucjp.json'                                              
npm WARN tar ENOENT: no such file or directory, open '/app/node_modules/.staging/flatpickr-17d30293/dist/flatpickr.min.css'                                                    
npm WARN tar ENOENT: no such file or directory, open '/app/node_modules/.staging/ajv-2fba4683/dist/ajv.min.js.map'                                                             
npm WARN tar ENOENT: no such file or directory, open '/app/node_modules/.staging/formiojs-e748ea40/dist/formio.contrib.min.js'                                                 
npm WARN tar ENOENT: no such file or directory, open '/app/node_modules/.staging/formiojs-14e0948d/dist/formio.contrib.min.js'                                                 
npm WARN tar ENOENT: no such file or directory, open '/app/node_modules/.staging/lodash-69176935/core.min.js'                                                                  
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: needle@2.2.0 (node_modules/fsevents/node_modules/needle):                                                                      
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, open '/app/node_modules/.staging/needle-436f24b5/package.json'                                
                                                                                                                                                                               
npm ERR! code E404                                                                                                                                                             
npm ERR! 404 Not Found: flatmap-stream@0.1.2                                                                                                                                   
                                                                                                                                                                               
npm ERR! A complete log of this run can be found in:                                                                                                                           
npm ERR!     /root/.npm/_logs/2018-11-27T20_41_29_757Z-debug.log                                                                                                               
ERROR: Service 'formio-api-app' failed to build: The command '/bin/sh -c cd /app &&            npm install' returned a non-zero code: 1                                        
make: *** [build] Error 1                                                                                                                                                      
```

**Comment by Netzvamp** [comment]
https://arstechnica.com/information-technology/2018/11/hacker-backdoors-widely-used-open-source-software-to-steal-bitcoin/

**Comment by randallknutson** [comment]
This is fixed in v1.37.5
